app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -3290,6 +3290,214 @@ def arg_defaults
       end
     end
 
+    class WinwoodLead < SonicPiSynth
+      def name
+        "Winwood Lead"
+      end
+
+      def introduced
+        Version.new(3,3,0)
+      end
+
+      def synth_name
+        "winwood_lead"
+      end
+
+      def on_start(studio, args_h)
+        args_h[:rand_buf] = studio.rand_buf_id
+      end
+
+      def doc
+        "A lead synth inspired by the Winwood songs from the early 80s.  Adapted for Sonic Pi from [Steal This Sound](https://raw.githubusercontent.com/supercollider/supercollider/develop/examples/demonstrations/stealthissound.scd). Published there under [GPL v3](https://www.gnu.org/licenses/gpl-3.0.en.html), so re-published under the same terms. The source code is available [on the Sonic Pi GitHub repository](https://github.com/sonic-pi-net/sonic-pi). Date of modification: 10.01.2021"
+      end
+
+      def arg_defaults
+        {
+          :note => 69,
+          :note_slide => 0,
+          :note_slide_shape => 1,
+          :note_slide_curve => 0,
+          :amp => 1,
+          :amp_slide => 0,
+          :amp_slide_shape => 1,
+          :amp_slide_curve => 0,
+          :pan => 0,
+          :pan_slide => 0,
+          :pan_slide_shape => 1,
+          :pan_slide_curve => 0,
+
+          :attack => 0.01,
+          :decay => 0,
+          :sustain => 0.9,
+          :release => 0.05,
+          :attack_level => 1,
+          :decay_level => 0.5,
+          :sustain_level => 0.5,
+
+          :cutoff => 119,
+          :cutoff_slide => 0,
+          :cutoff_slide_shape => 1,
+          :cutoff_slide_curve => 0,
+
+          :res => 0.8,
+          :res_slide => 0,
+          :res_slide_shape => 1,
+          :res_slide_curve => 0,
+          :lfo_width => 0.01,
+          :lfo_width_slide => 0,
+          :lfo_width_slide_shape => 1,
+          :lfo_width_slide_curve => 0,
+          :lfo_rate => 8,
+          :lfo_rate_slide => 0,
+          :lfo_rate_slide_shape => 1,
+          :lfo_rate_slide_curve => 0,
+          :seed => 0,
+        }
+      end
+      
+      def specific_arg_info
+        {
+          :seed =>
+          {
+            :doc => "Seed value for rand num generator used for the phase offset of the triangle low-frequency oscillator (LFO)",
+            :modulatable => false
+          },
+          :lfo_width =>
+          {
+            :doc => "Width of the low-frequency oscillator (LFO) which determines how wide base tones oscillate around their base frequencies",
+            :modulatable => true
+          },
+          :lfo_rate =>
+          {
+            :doc => "Width of the low-frequency oscillator (LFO) which determines how fast base tones oscillate around their base frequencies",
+            :modulatable => true
+          },
```

Comment:
If we want `lfo_rate:` to be slidable, and related `*_slide` opts visible in the docs, we'll need to add extra information here.
(How it works is that the metadata is constructed by merging `default_arg_info` (which holds standard opt hashes common to many synths/fx) with the `specific_arg_info` of each synth/fx. Since `lfo_rate:`, `lfo_rate_slide:`, etc are not part of the `default_arg_info`, we need to add hashes for them in this `specific_arg_info`).
```suggestion
          },
          :lfo_rate_slide =>
          {
            :doc => "Amount of time (in beats) for the rate of the low-frequency oscillator (LFO) to change. A long slide value means that the rate takes a long time to slide from the previous speed to the new speed. A slide of 0 means that the rate instantly changes to the new speed.",
            :validations => [v_positive(:lfo_rate_slide)],
            :modulatable => true,
            :bpm_scale => true
          },
          :lfo_rate_slide_shape =>
          {
            :doc => "Shape of curve for lfo_rate_slide  0: step, 1: linear, 2: exponential, 3: sine, 4: welch, 5: custom (use *_curve opt), 6: squared, 7: cubed, 8: hold",
            :validations => [v_one_of(:lfo_rate_slide_shape, [0, 1, 2, 3, 4, 6, 7, 8])],
            :modulatable => false
          },
          :lfo_rate_slide_curve =>
          {
            :doc => "Control curvature for lfo_rate_slide. Only used if *_slide_shape is 5. 0 means linear, positive and negative numbers curve the segment up and down.",
            :modulatable => false
          },
```

Reply:
(Note that this same update to the `specific_arg_info` also applies to other opts in your synths - basically any that do not appear in `default_arg_info`).

Reply:
Maybe I got that wrong, but after compilation that looked quite consistent with other options' slide params on my box. The code seemed to indicate to me that if a ``_slide`` param is found (like ``lfo_rate_slide`` for ``lfo_rate``) then these options are not show directly. Rather a yellow indicator appears in the main param ("Has slide options to shape changes ") and the "Slide Options" section is pulled in. This section appears at the bottom of the documentation. Hence, I understood that creating documentation for ``lfo_rate_slide`` and the like is not necessary. Am I wrong? (If a screenshot helps, just let me know.)

Reply:
🤦‍♂️ Sorry, you're completely correct - been a while since I've worked with slide opt metadata 😂